docker-compose.yml

```diff
@@ -2,6 +2,9 @@ version: '2'
 services:
   portal:
     build: .
+    environment:
+      - LIFERAY_WEB_PERIOD_SERVER_PERIOD_PROTOCOL=http
```

Comment:
Hey @sammonsjl, is the `http` protocol required for this example docker-compose file? I mean, `http` is the default value in portal.properties.